### PR TITLE
Rajesh-ppp_philippines

### PR DESCRIPTION
**Pull request by RajeshNN**
Prepared scraper that returns pandas dataframe of uncleaned data. Submitting for initial review.

### CODE REVIEW

**Comment by KayvanShah1** [comment]
We will soon schedule a technical discussion for first progress submission

File: PPP-Philippines/src/dependencies/scraping/scraper.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+
+
+class MyClass:
```
Comment (KayvanShah1) [inline PPP-Philippines/src/dependencies/scraping/scraper.py]: Change the class name to something sensible

File: PPP-Philippines/src/dependencies/scraping/scraper.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        return df
+
+    def run(self):
+        """Load data, do_something and finally save the data"""
+        links = self.extract_links()
+        return self.load_data(links)
+
+
+if __name__ == "__main__":
+    config = {'count': None}
```
Comment (KayvanShah1) [inline PPP-Philippines/src/dependencies/scraping/scraper.py]: Do you think only these parameters or attributes are to be specified in this config?

File: PPP-Philippines/src/dependencies/scraping/scraper.py (hunk 1 above)
Comment (KayvanShah1) [reply PPP-Philippines/src/dependencies/scraping/scraper.py]: Pretty unclear from where the data is loaded and then saved to

**Review by KayvanShah1 — COMMENTED** [review]
Please make sure you go through the data standards and requirements to write scraping and processing steps.

**Comment by KayvanShah1** [comment]
Please can you also specify your full in this thread so that we can also refer you to your application submitted via Internshala

File: PPP-Philippines/src/dependencies/scraping/scraper.py (hunk 2 above)
Comment (RajeshNN) [reply PPP-Philippines/src/dependencies/scraping/scraper.py]: I have added 'cols' as another parameter to draw only specific fields at the scraping level itself.

File: PPP-Philippines/src/dependencies/scraping/scraper.py (hunk 1 above)
Comment (RajeshNN) [reply PPP-Philippines/src/dependencies/scraping/scraper.py]: I was not saving intermediate data earlier. Made the changes to incorporate that. Also changed the names of classes.

**Comment by RajeshNN** [comment]
If you meant my full name, it is Rajesh. I am from Delhi.

**Pull request closed without merging**